images/build/Dockerfiles/vso.focal.Dockerfile

```diff
@@ -106,10 +106,10 @@ RUN set -ex \
     && DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 \
 	&& NUGET_PACKAGES="$nugetPackagesDir" \
     && . $buildDir/__dotNetCoreSdkVersions.sh \
-    && DOTNET_SDK_VER=$DOT_NET_CORE_21_SDK_VERSION \
+    && DOTNET_SDK_VER=$DOT_NET_CORE_31_SDK_VERSION \
        INSTALL_PACKAGES="true" \
        $imagesDir/build/installDotNetCore.sh \
-    && DOTNET_SDK_VER=$DOT_NET_CORE_31_SDK_VERSION \
+       && DOTNET_SDK_VER=$DOT_NET_60_SDK_VERSION \
```

Comment:
nit: the alignment of this line to the others that start with `&&` is now off -- would you mind lining it back up?